src/shared/components/lollipopMutationPlot/lollipopMutationPlotControls.scss

```diff
@@ -0,0 +1,22 @@
+.lollipop_mutation_plot__controls {
```

Comment:
This file contains partially duplicated code from `components/oncoprint/controls/styles.scss`, we may want to unite them at some point